### DISCUSSION TITLE
Error: no matching neighboring interior cell

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#591] [Q&A]
when I run the example of tpv105, I encouter this problem.
![image](https://user-images.githubusercontent.com/104721358/176430675-01fd5c1d-2782-4a33-82af-45576a171450.png)

**Comment by sebwolf-de** [comment]
Could not reproduce due to missing information.

**Comment by daisy20170101** [comment]
How did you generate the mesh?

**Reply by smallriver666** [reply]
> How did you generate the mesh?

I copy it from https://github.com/SeisSol/Examples/tree/master/tpv105, and generate the mesh by `./generating_the_mesh.sh `, my seissol version is hsw_6

**Comment by AliceGabriel** [comment]
Hi @chaoxinchen, 
For constructively engaging with the SeisSol team (consisting of PhD students and PostDocs) I ask you to follow the template provided when creating a new issue. I copy it below for your convenience.
I would also like to point you to our Code of Conduct here: https://github.com/SeisSol/SeisSol/blob/master/CODE_OF_CONDUCT.md
Please note, that issues are meant to either contain *documented* bug reports or feature requests. If you would like to simply chat please use the discussion forum instead.
 
Alice Gabriel

**Describe the bug**
A clear and concise description of what the bug is. Tell us what the code actually does.

**Expected behavior**
A clear and concise description of what you expected to happen. Tell us what you want the code to do.

**To Reproduce**
Steps to reproduce the behavior:
1. Which version do you use? Provide branch and commit id.
2. Which build settings do you use? Which compiler version do you use? 
3. On which machine does your problem occur? If on a cluster: Which modules are loaded?
4. Provide parameter/material files.

**Screenshots/Console output**
If you suspect a problem in the numerics/physics add a screenshot of your output.

If you encounter any errors/warnings/... during execution please provide the console output.

**Additional context**
Add any other context about the problem here.

**Reply by smallriver666** [reply]
> ok, I'm sorry for not describing my problem clearly.
